internal/super/generate.go

```diff
@@ -122,64 +136,60 @@ func generateScript(
 	_ flowkit.Services,
 	state *flowkit.State,
 ) (result command.Result, err error) {
-	return generateNew(args, "script", generateFlags.Directory, logger, state)
-}
-
-func addCDCExtension(name string) string {
-	if strings.HasSuffix(name, ".cdc") {
-		return name
+	options := GeneratorOptions{
+		Directory: "cadence",
+		State:     state,
+		Logger:    logger,
 	}
-	return fmt.Sprintf("%s.cdc", name)
+	generator := NewGenerator(options)
+	err = generator.Create(TemplateMap{"script": args[0]})
+	return nil, err
 }
 
-func stripCDCExtension(name string) string {
-	return strings.TrimSuffix(name, filepath.Ext(name))
+// TemplateMap defines a map of template types to their specific names
+type TemplateMap map[string]string
+
+type GeneratorOptions struct {
+	Directory   string
+	State       *flowkit.State
+	Logger      output.Logger
+	DisableLogs bool
```

Comment:
Could this be handled by a NOOP/nil logger?

Reply:
Will that this in when we have support